docs/RFCS/node_monitor.md

```diff
@@ -0,0 +1,69 @@
+- Feature Name: Node Monitor
+- Status: draft
+- Start Date: 2015-08-20
+- RFC PR:
+- Cockroach Issue: #2149 and #620
+
+# Summary
+
+Add a new `Monitor` service on each node that monitors all the other nodes and
```

Comment:
I just didn't want to use the term status again, since that's very involved with the status server.  But monitor is also used all over too.
Perhaps going with `NodePool` might be better.  Which would make sense if we add some of the decision making logic to it.